gr-utils/python/modtool/core/rm.py

```diff
@@ -78,12 +78,12 @@ def _remove_cc_test_case(filename=None, ed=None):
                                     r'\$\{CMAKE_CURRENT_SOURCE_DIR\}/%s' % filename,
                                     to_ignore_start='APPEND test_{}_sources'.format(self.info['modname']))
                     self.scm.mark_file_updated(ed.filename)
-            elif self._info['version'] == '38':
+            elif self.info['version'] == '38':
                 (base, ext) = os.path.splitext(filename)
                 if ext == '.cc':
                     ed.remove_value(
                         'list', filename,
-                        to_ignore_start='APPEND test_%s_sources' % self._info['modname'])
+                        to_ignore_start='APPEND test_%s_sources' % self.info['modname'])
```

Comment:
It would be nice if we could change the string formatting here as well.

Reply:
Done